DevTools/HackStudio/main.cpp

```diff
@@ -711,7 +722,18 @@ static int main_impl(int argc, char** argv)
         remove_current_terminal_action->set_enabled(is_remove_terminal_enabled());
     };
 
-    g_open_file = open_file;
+    on_action_tab_change = [&]() {
+        update_actions();
+        auto git_widget = s_action_tab_widget->active_widget();
+        if (!git_widget)
+            return;
+        if (strcmp(git_widget->class_name(), "GitWidget") != 0)
+            return;
```

Comment:
This looks a little awkward. How about `if (StringView(git_widget->class_name()) != "GitWidget")`
Hrm.. I guess that's also a little awkward :)

Reply:
I'd personally do it the other way around:
```c++
if (StringView { "GitWidget" } != git_widget->class_name())
```

Reply:
Perhaps we should have a UDL for `StringView`'s?

Reply:
> I'd personally do it the other way around:
> 
> ```c++
> if (StringView { "GitWidget" } != git_widget->class_name())
> ```

sure this reads nicely